### PR TITLE
Fix ROCm-aware MPI extension

### PR DESCRIPTION
**Pull request by utkinis**
This PR fixes a bug introduced with a recent update of AMDGPU.jl

### CODE REVIEW

**Comment by luraess** [comment]
We 

> Does this require a compat bounds update? How about a regression test?

Yes, as already done in #755. But let's do it here.

**Comment by utkinis** [comment]
@giordano I've updated combat bounds for AMDGPU.jl. There is no other functionality to do the regression testing against, this change just fixes the conversion of ROCArray to a Julia pointer, which didn't exist in AMDGPU prior to 0.5.7

**Comment by luraess** [comment]
Can we merge this and patch release?

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 *.mem
 Manifest.toml
 LocalPreferences.toml
+
+.vscode/
```
Comment (simonbyrne) [inline .gitignore:6]: This should go in your own excludes file, see https://git-scm.com/docs/gitignore

Comment (simonbyrne) [inline .gitignore:6]:
```suggestion
```

**Comment by luraess** [comment]
Thanks!

**Pull request merged**